db-service/test/assocs/unmanaged-assocs.test.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-console */
 const cds = require('@sap/cds')
+cds.test.in(__dirname)
```

Comment:
I have seen this issue before. It happens because `cds-test` now lazy loads `cds.test` and therefor `require` is not defined until `cds.test` is accessed once. One way to solve this issue is by using the extended `cds.test` definition in `test/cds.js`.

like:
```suggestion
const cds = require('../../../test/cds')
```